salt/_states/metalk8s_cordon.py

```diff
@@ -1,10 +1,5 @@
 from __future__ import absolute_import
 
-import logging
-
-
-log = logging.getLogger(__name__)
-
 
 def _node_set_unschedulable(name, value, **kwargs):
```

Comment:
It's sad how we make two API calls for those state methods, while [K8s API supports a dry-run mode](https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run)... Not worth tackling in this PR, but I think I'll open a tracking issue for it.

Reply:
If we use dry-run we will also make two API calls ? I don't get the point ? 

Also why we would like to do dry run only to check if we need to change the object

Reply:
My point was to not check the object at all. However, since we want to compute `changes` for Salt anyway, we'll need the original object (unlucky that K8s is not able to return the changes itself :disappointed:), so indeed, no point in using dry-run.